From 6c289a76ca9134c081c24963270140373e326552 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Mon, 27 Aug 2018 11:30:47 -0700
Subject: [PATCH] Fix build with boost 1.68.0

From the boost 1.68.0 release notes:

Breaking change: sha1 detail namespace header redirection for backwards
compatibility was removed (https://github.com/boostorg/uuid/pull/69).

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 src/libcmis/xml-utils.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libcmis/xml-utils.cxx b/src/libcmis/xml-utils.cxx
index 1e94ce3..1c8e13e 100644
--- a/src/libcmis/xml-utils.cxx
+++ b/src/libcmis/xml-utils.cxx
@@ -31,7 +31,7 @@
 #include <sstream>
 #include <stdlib.h>
 
-#include <boost/uuid/sha1.hpp>
+#include <boost/uuid/detail/sha1.hpp>
 #include <curl/curl.h>
 
 #include "xml-utils.hxx"
-- 
2.18.0

